wled00/wled_server.cpp

```diff
@@ -482,7 +482,17 @@ void initServer()
       return;
     }
 
-    if(handleSet(request, request->url())) return;
+    if (request->url().indexOf("win") == 0) {
+      if (handleHttpApi(request->url().substring(3))) {
```

Comment:
Is it 3 or is it 4?
I would think 4 as you do not compare `&` in `handleHttpApi()` or there is no need to add equality to comparisons there.

Reply:
The leading '&' is intentionally left on as some of the test strings in `handleHttpApi()` have a leading "&", eg. `pos = req.indexOf(F("&S="));`.  I'm not really sure why some do and some don't -- I should probably just clean that up too while I'm in there.

Reply:
Actually, given that, I'm going to convert this to draft for now and do some more testing.  I think there's some corner cases where this code will differ from the original.